application/controllers/customer/Cart.php

```diff
@@ -11,7 +11,7 @@ public function index(){
         //$this->load->model('Customer_model')
         $this->check_is_login('customer');
         $data['css'] = $this->load->view('includes/css.php', NULL, TRUE);
-        $data['navbar'] = $this->load->view('includes/user/navbar', NULL, TRUE);
+        $data['navbar'] = $this->load->view('includes/shop/header_logged', NULL, TRUE);
```

Comment:
ini aku mikir buat nampilin navbar kalau user udah login, cmn msh gatau cara otomatisnya yang pakai javascript dan kayaknya yang paling update filenya namnaya header_collections_logged.php

Reply:
ini mksdnya gmna ya?

Reply:
ini buat @samlrnt , lg mnt tolong sam cariin cara buat ganti header kalau user udh logged in, kan nnti tulisan sign in & sign up nya harusnya jadi ilang & diganti shopping cart, profile, sm kayak "Hello [username]" gitu